SPECS/dbus-python/dbus-python.spec

```diff
@@ -0,0 +1,318 @@
+Summary:        D-Bus Python Bindings
+Name:           dbus-python
+Version:        1.2.16
+Release:        3%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.freedesktop.org/wiki/Software/DBusBindings/
+Source0:        https://dbus.freedesktop.org/releases/%{name}/%{name}-%{version}.tar.gz
+# borrow centos7 patch to use sitearch properly
+Patch0:         0001-Move-python-modules-to-architecture-specific-directo.patch
+# https://bugzilla.redhat.com/show_bug.cgi?id=1788491
+Patch1:         python39.patch
+
+BuildRequires:  autoconf-archive
+BuildRequires:  automake
+BuildRequires:  dbus-devel
+BuildRequires:  dbus-glib-devel
+BuildRequires:  libtool
+
+%description
+D-Bus python bindings for use with python programs.
+
+%package -n python3-dbus
+Summary:        D-Bus bindings for python3
+
+%{?python_provide:%python_provide python3-dbus}
+BuildRequires:  python3-devel
+# for py3_build
+BuildRequires:  python3dist(setuptools)
+
+%description -n python3-dbus
+%{summary}.
+
+%package devel
+Summary:        Libraries and headers for dbus-python
+
+%description devel
+Headers and static libraries for hooking up custom mainloops to the dbus python
+bindings.
+
+%prep
+%autosetup -p1
+
+# For new arches (aarch64/ppc64le), and patch0
+autoreconf -vif
+
+%build
+%set_build_flags
+%py3_build
+%configure PYTHON="%{__python3}"
+%make_build
+
+%install
+%py3_install
+%make_install
+
+# unpackaged files
+rm -fv  %{buildroot}%{python3_sitearch}/*.la
+rm -rfv %{buildroot}%{_docdir}/dbus-python/
+
+%check
+make check -k || (cat test-suite.log && false)
+
+%files -n python3-dbus
+%doc NEWS
+%license COPYING
+%{python3_sitearch}/*.so
```

Comment:
Shouldn't this so be versioned?

Reply:
We'll get to that in a bulk fix for our specs. Won't fix for now.